pkgs/tools/security/pynitrokey/default.nix

```diff
@@ -4,47 +4,50 @@
 , nrfutil
 , libnitrokey
 , nix-update-script
+, installShellFiles
 }:
 
 with python3Packages;
 
 buildPythonApplication rec {
   pname = "pynitrokey";
-  version = "0.4.40";
+  version = "0.4.41";
   format = "pyproject";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-Hu+8UooDzv4GhkWt0sCckQQyHjWn4V/zt2ADlVCoHmk=";
+    hash = "sha256-QszxzZ2I/Bc7stcUh02w6JW/UlF9l77ZyaeWP9TXhPY=";
   };
 
   propagatedBuildInputs = [
     certifi
     cffi
     click
-    click-aliases
     cryptography
     ecdsa
     frozendict
     fido2
     intelhex
     nkdfu
-    nrfutil
     python-dateutil
     pyusb
     requests
-    semver
     spsdk
     tqdm
-    urllib3
     tlv8
     typing-extensions
+    pyserial
+    protobuf
+    click-aliases
+    semver
+    nethsm
     importlib-metadata
   ];
 
   nativeBuildInputs = [
     flit-core
     pythonRelaxDepsHook
+    installShellFiles
   ];
 
   # FIXME: does pythonRelaxDepsHook not work for pypaBuildHook + flit-core?
```

Comment:
If I understand correctly, `pythonRelaxDepsHook` cannot work because it is run during `postBuild` (see [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/python/hooks/python-relax-deps-hook.sh#L110)).

Reply:
Thank you for pointing this out, I was confused since I read the same thing when looking at nitrokey-app2, I now realise that it is `pypaBuildFlags = [ "--skip-dependency-check" ];` causing pynitrokey to work anyway.